src/api/app/views/webui2/webui/user/edit.html.haml

```diff
@@ -0,0 +1,34 @@
+- @pagetitle = 'Edit User Data'
+
+.container
+  .card
+    .card-body
+      %h2
+        Editing User #{@displayed_user.login}
+      .row
+        .col-md-6
+          = form_for(@displayed_user, url: user_save_path, method: 'post') do |form|
```

Comment:
We need to let our users know which fields are required. For an example to see how we did it before, please have a look at #7017.